### DISCUSSION TITLE
Can you put notices in numbered lists with continued numbering?

### DISCUSSION THREAD

**Discussion by designatedsuccessor** [outline/outline#7957] [General]
Almost 100% of our docs are numbered lists.  Let's say I have a list of 5 steps:
1. Step 1
2. Step 2
3. Step 3
4. Step 4
5. Step 5

My understanding is that if I want there to be, say, an info notice between Step 2 and Step 3, I have to break the numbering between two lists: one before the notice and one after the notice.  The problem with this is that if the number of steps changes _before_ the notice, it is not automatically reflected in the list _below_ the notice.  Example:

1[.]  Step 1
2[.]  Step 2
3[.]  New Step 3
4[.]  New Step 4
[Notice goes here]
3[.]  Old Step 3
4[.]  Old Step 4

Is there a way to avoid this so that the two lists are actually one list and what goes below the notice automatically becomes Step 5 and Step 6?

EDIT: lol  Well, it appears Github does it the way I am suggesting because I had to put the brackets around the periods ([.]) to make it not automatically convert Old Step 3 and Old Step 4 to Step 5 and Step 6.

**Comment by tommoor** [comment]
You can start a list at any number by typing something like `2. ` – this allows you to stop a list, add some other block content, and then restart the list further down the document. I hope that works.


https://github.com/user-attachments/assets/6ae8e95c-e47b-4d90-b9bb-1e8ab45d5334

**Reply by designatedsuccessor** [reply]
That's actually the problem I'm describing. When I do exactly that, the top list and bottom list are "disconnected." And that means that if numbering on the top list changes, numbering in the bottom list has to be manually updated. It isn't so bad if you have only a few notices, but it's awful in longer documents.

**Reply by designatedsuccessor** [reply]
I also noticed that you changed the title of this thread.  I deliberately did not make it that title because that isn't what I was asking.

**Reply by tommoor** [reply]
I didn't touch the title of the thread.

The other alternative is to create newlines in a list item with `shift`+`enter`. Currently you would then have to drag a notice or other element into those empty lines.
